### DISCUSSION TITLE
The README

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3986] [General]
Our README is huge, the problem is that nobody reads it when upgrading.
I worked to write better and more comprensive release notes for every new release because it seems to be more useful.

The problem is that lately we didn't add to the README the "difference between v19 and v20" and also "difference between v20 and v21" (this was reported to be by @sreichel).
But does it make sense to keep adding stuff to the README? lately we merged a lot, the list would be huge.

To me, some parts of the README could/should be removed, some other could be "collapsible" (like the installation methods).

I also think the full list of contributors with all the pictures should go, it's some MegaBytes for every page refresh, there's a link to the list of contributors on the right sidebar and to me makes the page too messy because of the many colors, links etc...

What do you think?

**Comment by pquerner** [comment]
Collapsing and moving details into their own files seem like a good idea.
The wall of pictures could also go into its own file then.

**Comment by sreichel** [comment]
Closed with https://docs.openmage.org/

> The wall of pictures could also go into its own file then.

Then it also could be removed. No. ;)
